canmatrix/dbc.py

```diff
@@ -54,7 +54,7 @@ def format_float(f):
     s = str(f).upper()
     if s.endswith('.0'):
         s = s[:-2]
-
+        s = '%.7g' % float(s)
```

Comment:
Given that everything is (usually?) a decimal, perhaps we could format this stuff directly as desired rather than back and forth through string and float?

Reply:
I will remove this from my pull request. Formatting the max value for a signal in the DBC output may not be required. I can't reproduce the tool issue that I experienced earlier. 

A little over zealous on this part of my request.